modules/mobile/app/models/mobile/v0/adapters/facility_info.rb

```diff
@@ -4,29 +4,36 @@ module Mobile
   module V0
     module Adapters
       class FacilityInfo
-        def parse(facility, user, params)
-          user_location = params[:sort] == 'current' ? current_coords(params) : home_coords(user)
-
-          facility_parse(facility, user, user_location)
-        end
-
-        private
-
-        def facility_parse(facility, user, user_location)
+        def parse(facility:, user:, sort: nil, lat: nil, long: nil)
```

Comment:
Switched to keyword params because passing params didn't make as much sense for the new index endpoint and because the arg list was getting too long.